pallets/automation-time/src/mock.rs

```diff
@@ -598,6 +589,27 @@ pub fn last_event() -> RuntimeEvent {
 	events().pop().unwrap()
 }
 
+pub fn last_task_id() -> TaskIdV2 {
```

Comment:
This utility function looks good, however, could you add some comments to explain its purpose? Like, why using this approach is better than the existing code?